pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgres_types.go

```diff
@@ -61,4 +61,8 @@ type PostgresUserSpec struct {
 	// The secret name to generate user, password, connection info this PostgreSQL user.
 	// +optional
 	SecretName PostgresIdentifier `json:"secretName,omitempty"`
+
+	// Grant a specific user access to the public schema in every database where they already have permissions.
+	// +optional
```

Comment:
I recommend this comment with a pointer to the databases field to increase clarity: 

```
// Grant the user access to the public schema in each database listed under `databases`.
// +optional
```